packages/appeals-service-api/src/mappers/questionnaire-submission/has-mapper.js

```diff
@@ -0,0 +1,87 @@
+class HasQuestionnaireMapper {
+	mapToPINSDataModel(journeyResponse) {
+		return [
+			{
+				questionnaire: {
+					LPACode: journeyResponse.LPACode,
+					caseReference: journeyResponse.caseReference,
+					isAppealTypeAppropriate: journeyResponse.answers['correct-appeal-type'] ? true : false,
+					doesTheDevelopmentAffectTheSettingOfAListedBuilding: journeyResponse.answers[
+						'affects-listed-building'
+					]
+						? true
+						: false,
+					affectedListedBuildings: () => {
+						if (journeyResponse.answers['affects-listed-building']) {
+							return journeyResponse.answers['listed-building-number'];
+						}
+					},
+					siteWithinGreenBelt: journeyResponse.answers['green-belt'] ? true : false,
```

Comment:
Do we need to check the value of the answer here (e.g yes vs no instead of truthy?). If so could be another utility / helper function?